.azure-pipelines/ci.yml

```diff
@@ -20,3 +20,4 @@ variables:
 stages:
 - template: stage-lint.yml
 - template: stage-test.yml
+- template: stage-aarch64.yml
```

Comment:
this should probably go to `deploy.yml`

Reply:
@webknjaz Sorry for the confusion. stage-aarch64.yml(renamed as stage-test-aarch64) is for running pytest for aarch64, same as stage-test.yml is for x86, Mac and Windows.. I have renamed it to stage-test-aarch64 to avoid the confusion.
I am running this test in deploy.yml as well.